src/adapters/refract/transformResource.coffee

```diff
@@ -105,7 +104,8 @@ module.exports = (resourceElement, location, options) ->
 
       # In refract just here we have method and href
       resource.method = _.chain(httpRequest).get('attributes.method', '').contentOrValue().value()
-      resource.actionUriTemplate = _.chain(httpRequest).get('attributes.href', '').contentOrValue().value()
+      requestUriTemplate = _.chain(httpRequest).get('attributes.href', '').contentOrValue().value()
+      resource.actionUriTemplate = requestUriTemplate or transitionUriTemplate
```

Comment:
I got a little bit confused here. The old code was incorrect and incorrectly pulling a request's href and setting it on the resource.

When I looked here I didn't realise this was the case. I've updated the PR to correct this behaviour.